docusaurus/docs/reference/ha/certificates.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: Certificates
+sidebar_position: 20
+---
+
+# Controller Certificates
+
+For controllers to communicate and trust one another, they need certificates that have
+been generated with the correct attribute and relationships.
```

Comment:
This should expand on what SPIFFE IDs are and why they are used. I see them mentioned in the requirements section below, but adding the info to this first paragraph is useful. I believe many people will read the first paragraph and then skim the rest. Adding that sort of thing here helps them skim